src/tsickle_compiler_host.ts

```diff
@@ -6,7 +6,8 @@ import {convertDecorators} from './decorator-annotator';
 import {processES5} from './es5processor';
 import {ModulesManifest} from './modules_manifest';
 import * as sourceMapUtils from './source_map_utils';
-import {annotate, isDtsFileName} from './tsickle';
+import * as tsickle from './tsickle';
+import {isDtsFileName} from './tsickle';
```

Comment:
Yeah I struggled on this one.  I figured isDtsFileName itself is a pretty specific name (it doesn't even really belong in the tsickle module maybe?) but I guess it's trivial to change it later so I'll fix it.